core/src/main/java/com/novoda/noplayer/internal/exoplayer/ExoPlayerTwoImpl.java

```diff
@@ -150,17 +150,17 @@ public void seekTo(VideoPosition position) throws IllegalStateException {
     @Override
     public void stop() {
         reset();
+        listenersHolder.getStateChangedListeners().onVideoStopped();
```

Comment:
I guess this is sth else , that is not mentioned in the description of the PR 🤔  ?